pkg/web/context.go

```diff
@@ -69,7 +65,6 @@ func (ctx *Context) RemoteAddr() string {
 	if len(addr) > 0 {
 		if parsedIP := net.ParseIP(addr); parsedIP == nil {
 			// if parsedIP is nil we clean addr and populate with RemoteAddr below
-			ctx.logger.Warn("Received invalid IP address in request headers, removed for log forgery prevention")
```

Comment:
Would like to hear opinions on this one.

This error is functionally ignored anyways, only logged for reference.
Sure, it's nice to have this information, but not sure how important it actually is.

Didn't find a good way to preserve it without a side-channel aka logger though.